tests/unit/test_models/test_submodels/standard_submodel_unit_tests.py

```diff
@@ -0,0 +1,52 @@
+#
```

Comment:
why was this file added?

Reply:
I did not add this. It may be from an old branch. If you agree, I can remove it or leave it to you.